.github/workflows/conformance-aks.yaml

```diff
@@ -246,7 +246,7 @@ jobs:
 
       - name: Port forward Relay
         run: |
-          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
+          cilium hubble port-forward&
```

Comment:
Maybe worth pulling in https://github.com/cilium/cilium-cli/pull/390 at the same time?

Reply:
looks like it's blocked by this flake: https://github.com/cilium/cilium-cli/issues/399 😢

Reply:
Yes, but we don't need to wait for that commit to be merged in cilium-cli to pull it in cilium/cilium :-)

Reply:
Thanks, added in 086ff29af18d85c02848a2dced47683cec3cca97.